src/routes/people.route.js

```diff
@@ -1,107 +1,254 @@
 function peopleRoutes(database) {
+	const express = require("express"),
+		router = express.Router(),
+		helper = require("../helpers/helper");
 
-    const express = require('express'),
-        router = express.Router();
+	/**
+		GET people/all
+		used: get all people schema (by date).
+		status code: 
+			- 200 OK
+	*/
+	router.get("/all", async (_, res) => {
+		const allPeople = await database.getAllPeopleInfo();
+		// console.log(allPeople);
+		return res.send(allPeople);
+	});
 
+	/**
+		GET people/by_date/:date
+		used: get people schema by date.
+		status code: 
+			- 200 OK
+			- 406 no date param included in request.
+	*/
+	router.get("/by_date", "/by_date/:date", async (req, res) => {
```

Comment:
should be like	router.get(["/by_date", "/by_date/:date"], async (req, res) => {

Reply:
Thankyou!!